dashboard/lib/expired_deleted_account_purger.rb

```diff
@@ -87,13 +86,18 @@ def purge_expired_deleted_accounts!
   end
 
   private def expired_soft_deleted_accounts
-    soft_deleted_accounts.where 'deleted_at BETWEEN :start_date AND :end_date',
-      start_date: @deleted_after,
-      end_date: @deleted_before
+    user_ids_needing_manual_review = QueuedAccountPurge.all.map(&:user_id)
```

Comment:
Use `QueuedAccountPurge.pluck(:user_id)` instead of `.all.map(&:user_id)`.  Note the param is a symbol, not a proc, for `pluck`.

[Pluck](https://apidock.com/rails/ActiveRecord/Calculations/pluck) forms a sql query that only selects those attributes, whereas `all` will pull in every row in its entirety and then pick out the id in Rails.

Reply:
Good call, thanks!